docs/wifi.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_label: Configure Wi-Fi
+title: ''
+---
+
+
+### How to configure Wi-Fi
+
+:::info info
+The information on this page is just a specific use case of using cloud-config. For more generic info on how to create arbitrary files check our [cloud-config](cloud-config-reference.md) page
+:::
+
+Elemental Teal currently uses [NetworkManager](https://networkmanager.dev/) to manage network connections.
+
+In order to add Wi-Fi to your node, your registration should include a configuration in the [cloud-config](cloud-config-reference.md) section to write a
+`.connection` file so NetworkManager can connect to the Wi-Fi.
+
+A `.connection` file is a connection configuration file for NetworkManager.
+The connection files are stored under `/etc/NetworkManager/system-connections/` and can include ethernet, Wi-Fi, VPN and more.
+
+For example for a network with the SSID `testSSID` and the WEP key `1234567891` and using the interface `wlan0` you would write the following `.connection` file:
+
+```
+[connection]
+id=testConnection
+type=wifi
+interface-name=wlan0
+permissions=
```

Comment:
nit: empty values can be dropped

Reply:
Im not sure, can they? This is lifted directly from a working section of a real connection, and those values appear as is in the file. Im not sure if this is meant to mean something (like in this case, maybe its no permissions?) and overrides the default value?

Reply:
For example, in this case:

```
 An array of strings defining what access a given user has to this connection. If this is NULL or empty, all users are allowed to access this connection; otherwise users are allowed if and only if they are in this list. When this is not empty, the connection can be active only when one of the specified users is logged into an active session. Each entry is of the form "[type]:[id]:[reserved]"; for example, "user:dcbw:blah".
```

So it eneds to be empty so all users can use this connection....or NULL maybe?

Reply:
Yeah, good point, defaults in NetworkManager can be tricky (while they usually are documented, if any, in nm-settings).
For the permissions I think there couldn't be a default to a specific user, but I got it, better safe than sorry 😅 👍🏼